src/events/lavalink/rawLavalinkDestroy.ts

```diff
@@ -0,0 +1,19 @@
+import { Events } from '@lib/types/Enums';
+import { LavalinkDestroyEvent } from '@utils/LavalinkUtils';
+import { Event, EventStore } from 'klasa';
+
+export default class extends Event {
+
+	public constructor(store: EventStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			emitter: store.client.lavalink!,
+			event: 'destroy'
+		});
+	}
+
+	public run(payload: LavalinkDestroyEvent) {
+		const manager = this.client.guilds.get(payload.guildId)?.music || null;
```

Comment:
is the `|| null` required? Just using optional chaining it will be ` | undefined` which is already falsey. If not required then apply same to `rawLavalinkEvents`